### ISSUE TITLE
mac build?

### ISSUE THREAD

**Issue by iceboundflame** [chilipeppr/serial-port-json-server#11]
Is there a Mac pre-built binary available? The documentation says so but I don't see it on the releases page.

**Comment by chilipeppr** [comment]
Google plus page has it

**Comment by Fl0p** [comment]
http://chilipeppr.com/downloads/v1.86/serial-port-json-server-1.86_darwin_amd64.zip

**Comment by 01ivr3** [comment]
Anyone know where there's no macOS build for v1.95? ref: https://github.com/chilipeppr/serial-port-json-server/releases

**Comment by chilipeppr** [comment]
Yes, because Apple makes it impossible to let Go cross-compile from Linux
where I run the build script and I have no Apple machine to compile on. If
you could compile it and send me the binary, I'd be more than happy to post
it to the Github releases page.

**Comment by houda84** [comment]
I have an access to apple machine. just let me know what I should do.  Thanks in advance.

**Comment by chilipeppr** [comment]
That would be great. Maybe just email as an attachment to jlauer12@gmail.com
or post on drop box or another github repo?

**Comment by houda84** [comment]
Please find below repo for mac version. Working like charm on Catalina, followed your readme file. Please updated repo location (step6) and and add a a step 6.5 if you don't have  crypto ssh type "go get -u golang.org/x/crypto/..."  .

[https://github.com/houda84/Serial_Port_JSON_Server_Mac](url)

**Comment by chilipeppr** [comment]
Did you try the Darwin build in the 1.96 release? I actually forgot that Go does allow Mac cross-compiling now. https://github.com/chilipeppr/serial-port-json-server/releases

![image](https://user-images.githubusercontent.com/7904621/79645385-d98f1600-8163-11ea-8ece-b9d7c697fdb1.png)

**Comment by houda84** [comment]
It is executable but not functional. I get only [Last login: Sat Apr 18
21:23:48 on ttys000 ].

**Comment by jaclubjly** [comment]
Hi,
When I download "serial-port-json-server-1.96_darwin_amd64.zip", I get a TexEdit document, not a Unix Executable. (Mac OS 10.15.7 Catalina)
![Capture d’écran 2020-12-06 à 21 07 12](https://user-images.githubusercontent.com/75329459/101291241-9e7f4700-3807-11eb-8dbe-4a61aa326862.png)
I don't know what to do, thanks for your help.

**Comment by chilipeppr** [comment]
Do a chmod 777 to give it permissions to be to an executable file.

**Comment by jaclubjly** [comment]
Thank you for your quick response !
I can connect to 2 ports tty.usbmodem146201 and cu.usbmodem146201.
Both seam to work fine, is one better than the other ?
